Sources/LLVM/PassManager.swift

```diff
@@ -106,8 +106,6 @@ public enum FunctionPass {
   case scalarReplAggregates
   /// Replace aggregates or pieces of aggregates with scalar SSA values.
   case scalarReplAggregatesSSA
-  /// Tries to inline the fast path of library calls such as sqrt.
-  case simplifyLibCalls
```

Comment:
Why is this removed?

Reply:
The corresponding add pass function is a no-op as the pass has been removed.